### PR TITLE
Frontend home page

### PR DESCRIPTION
**Pull request by rajuTK9**
- React app has been initialized with TypeScript.
- Home page UI has been created.
- Home page has been made completely responsive.

Screenshot 2023-06-21 at 1 15 31 PM

image

### CODE REVIEW

File: client/public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
```
Comment (PatelNeelMahesh) [inline client/public/manifest.json]: Add your application name here.

File: client/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "client",
```
Comment (PatelNeelMahesh) [inline client/package.json]: Add your application name here

File: client/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```
Comment (PatelNeelMahesh) [inline client/README.md]: Remove default Readme generated by create-react-app and add details about components figma design in Readme.

**Review by PatelNeelMahesh — COMMENTED** [review]
You can merge pull request after making below changes.

File: client/package.json (hunk 2 above)
Comment (rajuTK9) [reply client/package.json]: Done

File: client/README.md (hunk 3 above)
Comment (rajuTK9) [reply client/README.md]: Done

**Pull request merged**